<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyDialog</class>
 <widget class="QDialog" name="MyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>参数设置</string>
  </property>
  <widget class="QLineEdit" name="txt4">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>290</y>
     <width>130</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>290</y>
     <width>129</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>290</y>
     <width>130</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>290</y>
     <width>129</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Save">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>450</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Cancle">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>450</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>450</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel  </string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>170</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>启用读卡器</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>打印模块１</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>170</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>启用打印机</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>200</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>打印模块２</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>200</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>启用读卡器</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_4">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>200</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>启用打印机</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>230</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>打印模块３</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_5">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>230</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>启用读卡器</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_6">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>230</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>启用打印机</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>271</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>数据包</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rbtn3">
      <property name="text">
       <string>自动签收</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rbtn4">
      <property name="text">
       <string>手动签收</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>251</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>工作模式</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rbtn1">
      <property name="text">
       <string>自动</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rbtn2">
      <property name="text">
       <string>手动</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="ImageSwitch" name="solt1_switch" native="true">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>390</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_slotTitle1_4">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>390</y>
     <width>39</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>卡槽4</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_slotTitle1_3">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>390</y>
     <width>39</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>卡槽3</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_slotTitle1_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>390</y>
     <width>39</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>卡槽2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_slotTitle1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>390</y>
     <width>39</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>卡槽1</string>
   </property>
  </widget>
  <widget class="ImageSwitch" name="solt2_switch" native="true">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>390</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="ImageSwitch" name="solt3_switch" native="true">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>390</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="ImageSwitch" name="solt4_switch" native="true">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>390</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageSwitch</class>
   <extends>QWidget</extends>
   <header>control/imageswitch.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
